BPOR/NIHR.Infrastructure/Settings/EmailSettings.cs

```diff
@@ -7,6 +7,8 @@ public class EmailSettings : IValidatableObject
     {
         public const string SectionName = nameof(EmailSettings);
         public string FromAddress { get; set; }
+        public bool? Enabled { get; set; }
```

Comment:
Is this used anywhere?

Ideally we would want a production safe default, where it is normally enabled and we have to turn it off in lower environments.